libkbfs/path.go

```diff
@@ -46,21 +49,29 @@ func BuildCanonicalPath(pathType PathType, paths ...string) string {
 	return strings.Join(pathElements, "/")
 }
 
-// buildCanonicalPathForTlfName returns a canonical path for a tlf.
-func buildCanonicalPathForTlfName(public bool, tlfName CanonicalTlfName) string {
-	pathType := PrivatePathType
-	if public {
+func buildCanonicalPathForTlfType(t tlf.Type, paths ...string) string {
+	var pathType PathType
+	switch t {
+	case tlf.Private:
+		pathType = PrivatePathType
+	case tlf.Public:
 		pathType = PublicPathType
+	case tlf.SingleTeam:
```

Comment:
ugh, the link doesn't bring you to the correct place. I meant in the function `BuildCanonicalPath` in `libkbfs/path.go`